wp-cldr.php

```diff
@@ -33,20 +33,28 @@ function wp_cldr_menu() {
 }
 
 /**
+ * Loads the text domain for the plug-in. We call this only when the settings page is loaded since translations aren't needed any earlier.
+ */
+function wp_cldr_load_plugin_textdomain() {
+	load_plugin_textdomain( 'wp-cldr', false, basename( dirname( __FILE__ ) ) . '/languages/' );
```

Comment:
I blame https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#loading-text-domain which use the `dirname( __FILE__ )` :)